core4/api/v1/request/standard/template/help.html

```diff
@@ -1,283 +1,379 @@
-<script type="text/javascript">
-    $('body').scrollspy({
-        target: '#help-list'
+    </v-app>
+</div>
+<script src="https://bi.plan-net.com/cdn/assets/js/vue-assets_2.0.js"></script>
+<script>
+    new Vue({
+        delimiters: ['[[', ']]'],
+        el: '#app',
+        vuetify: new Vuetify({
+            icons: {
+                iconfont: 'mdiSvg', // 'mdi' || 'mdiSvg' || 'md' || 'fa' || 'fa4'
+            },
+            theme: window.VUETIFY_THEME
+        }),
+        data() {
+            return {
+                tabs: 0,
+                subtitle: '{{subtitle}}',
+                qual_name: '{{qual_name}}',
+                title: '{{title}}',
+                author: '{{author}}',
+                protected: '{{protected}}',
+                rsc_id: '{{rsc_id}}',
+                container: '{{container}}',
+                icon: '{{icon}}',
+                enter_url: '{{enter_url}}'
+            }
+        },
+        computed: {
+            args: function () {
+                var result = {
+                        data: {},
+                        template: '{{args}}'
+                    },
+                    regDict= {
+                        "&#39;": `"`,
+                        "True": true,
+                        "False": false,
+                        "None": null
+                    }
+
+                try {
+                    const re = new RegExp(`(${ Object.keys(regDict).join('|') })`,'gi')
+                    const clean_str = result.template.replace(re, m => regDict[m])
+
+                    result.data = JSON.parse(clean_str)
+                    result.template =  JSON.stringify(result.data, undefined, 4)
+                } catch (e) {
+                    console.warn(`Error with a parsing string ${result}`)
+                    console.error(e)
+                }
+
+                return result;
+            },
+            isCoreStaticFileHandler: function () {
+                const qual_name = '{{qual_name}}'
+                return qual_name.split(".").slice(-1)[0] === 'CoreStaticFileHandler'
+            }
+        },
+        mounted() {
+            document.querySelector('body').style.opacity = 1;
+            window.setTimeout(function(){
+                const rubric = document.querySelectorAll('.rubric');
+                const methods =  ["GET", "POST", "DELETE", "PUT"];
+                for (var i = 0; i < rubric.length; i++) {
+                    var item = rubric[i];
+                    const text = item.textContent.split(' ')[0]
+                    if(methods.indexOf(text) > -1){
+                        item.innerHTML = text + ' <span class="code" style="margin-left: 90px;">' + item.textContent.replace(text, '') + '</span>' 
+                    }
+                }
+                [].map.call(document.querySelectorAll('.introduction-desc pre.literal-block'), function(element) {
```

Comment:
You can also use forEach - document.querySelectorAll('.introduction-desc pre.literal-block').forEach((elem)=> {...})